andela/ah-kgl-avengers-frontend
# #166727757 Show user feed on home page

### What does this PR do?
- Get user feeds

### Description of Task to be completed?
- Get user feeds and display them on home page 
- Save article featured image when creating/updating an article

### How should this be manually tested?

```sh
git pull origin
git checkout ft-get-article-feed-166727757
npm install
npm start
```
- Run the following code and then navigate to `localhost:8000/` to see the feeds
- Login on `localhost:8000//login`and then go to `localhost:8000/new-post` to create a new post
- Add text and then add an image
- Publish the article
- The best way to see that the first image is the featured image is to inspect the response in the dev tools. You can also go on the feeds page that same image will be shown for the article.

Run local tests:
```sh
git pull origin
git checkout ft-get-article-feed-166727757
npm install
npm test
```

### Any background context you want to provide?
- You need an activated account to create an article
- The article editor has a bug❗️❗️Before adding an image you  need to have text in the editor
   To be fixed

### What are the relevant pivotal tracker stories?
[Get article feed #166727757](https://www.pivotaltracker.com/n/projects/2321840/stories/166727757)

### Screenshots 
<img width="1209" alt="Screen Shot 2019-06-20 at 12 59 14" src="https://user-images.githubusercontent.com/24508902/59844458-3ad73480-935b-11e9-9274-bc423d8df12b.png">


### Questions:
None
